src/main/java/ru/spbstu/reader/dto/VcfRecord.java

```diff
@@ -0,0 +1,53 @@
+package ru.spbstu.reader.dto;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Map;
+import java.util.Objects;
+
+@Getter
+@AllArgsConstructor
+public class VcfRecord {
+    private final String chrom;
+    private final int pos;
+    private final String id;
+    private final Allele ref;
+    private final Allele alt;
+    private final Map<String, Object> info;
```

Comment:
Let's not use htsjdk classes because we don't want to tie on a specific library.
We can simplify it using `String` for chrom, ref, alt.
Also I suggest to use `Map<String, String>` for info field.

Reply:
+1. Try to avoid using htsjdk classes. Object is also very inconvenient to work with.